src/agent/coverage/src/block.rs

```diff
@@ -57,12 +57,39 @@ impl CommandBlockCov {
         self.modules.iter()
     }
 
+    /// Total count of covered blocks across all modules.
+    pub fn covered(&self) -> u64 {
+        self.modules.values().map(|m| m.covered()).sum()
+    }
+
+    /// Total count of known blocks across all modules.
+    pub fn features(&self) -> u64 {
```

Comment:
I think `known_blocks` would be a better name.

Reply:
Will switch. This more-abstracted name was chosen for consistency with OneFuzz, but I agree that `known_blocks` would be clearer (and this doesn't impact user-facing serialization, so we are free to diverge).